### PR TITLE
[Jetcaster] Implement home UX changes

### PR DESCRIPTION
**Pull request by chrisbanes**

### CODE REVIEW

File: Jetcaster/app/src/main/java/com/example/jetcaster/ui/home/Home.kt [hunk 1]
```diff
@@ -104,36 +105,34 @@ fun Home() {
...
-                onClick = { /* TODO: Open account? */ },
-                modifier = Modifier.align(Alignment.CenterStart)
-                    .padding(start = 8.dp)
-            ) {
-                Icon(Icons.Default.AccountCircle)
+    TopAppBar(
+        title = {
+            Row {
+                Image(asset = vectorResource(R.drawable.ic_logo))
+                Icon(
```
Comment (nickbutcher) [inline Jetcaster/app/src/main/java/com/example/jetcaster/ui/home/Home.kt:115]: Why use `Icon` here? Feels like `Image` would be more fitting as you're not using the default sizing?

File: Jetcaster/app/src/main/java/com/example/jetcaster/ui/home/Home.kt [hunk 2]
```diff
@@ -179,38 +178,38 @@ fun HomeContent(
...
+                    .verticalGradientScrim(
+                        color = MaterialTheme.colors.primary.copy(alpha = 0.38f),
+                        startYPercentage = 1f,
+                        endYPercentage = 0f
+                    )
+            ) {
+                val appBarColor = MaterialTheme.colors.surface.copy(alpha = 0.87f)
+
+                // Draw a scrim over the status bar which matches the app bar
+                Spacer(Modifier.background(appBarColor).fillMaxWidth().statusBarsHeight())
```
Comment (nickbutcher) [inline Jetcaster/app/src/main/java/com/example/jetcaster/ui/home/Home.kt:192]: This makes me want a `.statusBarSize` modifier.

Comment (chrisbanes) [reply Jetcaster/app/src/main/java/com/example/jetcaster/ui/home/Home.kt:192]: Yeahhhh, good shout.

**Comment by nickbutcher** [comment]
Code lg but let's land this in `dev_alpha04` once that's cut

**Comment by chrisbanes** [comment]
No idea what googlebot is complaining about.

File: Jetcaster/app/src/main/java/com/example/jetcaster/ui/home/Home.kt (hunk 1 above)
Comment (chrisbanes) [reply Jetcaster/app/src/main/java/com/example/jetcaster/ui/home/Home.kt:115]: It gives the automatic `contentColor`. Not a huge win, but good enough imo.

**Review by nickbutcher — APPROVED** [review]

**Pull request merged**